### DISCUSSION TITLE
Add ability to specify another model name

### DISCUSSION THREAD

**Discussion by Hirishiolo** [reliese/laravel#299] [Ideas]
Example:

`php artisan code:models --table=users --model-name=Utente`

It will create model "Utente"

thanks in advance

**Comment by Benjaminhu** [comment]
@Hirishiolo This feature supported by [config](https://github.com/reliese/laravel/blob/v1.x/config/models.php):
```php
        /*
        |--------------------------------------------------------------------------
        | Model Names
        |--------------------------------------------------------------------------
        |
        | By default the generator will create models with names that match your tables.
        | However, if you wish to manually override the naming, you can specify a mapping
        | here between table and model names.
        |
        | Example:
        |   A table called 'billing_invoices' will generate a model called `BillingInvoice`,
        |   but you'd prefer it to generate a model called 'Invoice'. Therefore, you'd add
        |   the following array key and value:
        |     'billing_invoices' => 'Invoice',
        */

        'model_names' => [

        ],
```

**Comment by Hirishiolo** [comment]
yes, thank you very much. It would be fantastic if that can be used inline as I suggested!
Good job!

**Reply by Benjaminhu** [reply]
@Hirishiolo I don't fully understand why it needs to be specified inline on the command line? Does its name change sometimes? If it's specified in the config, it can be committed and it's guaranteed that the name will be created with the alias specified there.

**Reply by Hirishiolo** [reply]
Simply for convenience, I prefer using it "live" and don't commit nothing
